sys-kernel/zen-sources/zen-sources-6.5.5.ebuild

```
@@ -0,0 +1,56 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+ETYPE="sources"
+K_WANT_GENPATCHES="base extras"
+K_GENPATCHES_VER="7"
+K_SECURITY_UNSUPPORTED="1"
+K_NOSETEXTRAVERSION="1"
+
+inherit kernel-2 unpacker
+detect_version
+detect_arch
+
+KEYWORDS="~amd64 ~arm64 ~x86"
+HOMEPAGE="https://github.com/zen-kernel"
+IUSE=""
+
+# Needed for zstd compression of the patch
+# Is this actually needed with the unpacker eclass?
+BDEPEND="app-arch/zstd"
```

To answer your q, not if we do:
```suggestion
BDEPEND="$(unpacker_src_uri_depends)"
```